style.css

```diff
@@ -0,0 +1,53 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
Delete the body or put some styles in it .

Reply:
- [ ] please give a reason for this change u demanded. why will it be necessary to delete the body:?

Reply:
Done